src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPool.Portable.cs

```diff
@@ -31,6 +31,9 @@ public static partial class ThreadPool
             AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.EnableWorkerTracking", false);
 #endif
 
+        internal static readonly bool EnableDispatchAutoReleasePool =
+            AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.EnableDispatchAutoreleasePool", false);
```

Comment:
Can this just be in `ThreadPool.cs`? It seems to be the same across coreclr and non-coreclr.

Reply:
Funny thing, `ThreadPool.cs` doesn't actually contain the ThreadPool type. It only has the `ThreadPoolWorkQueue` type and nested types in it.